### ISSUE TITLE
idea: consider an alternate way to specify cell formatting

### ISSUE THREAD

**Issue by JessicaMulein** [Digital-Defiance/OpenBook#7]
currently we have the support for = formulas and =$ formulas which are formatted as currencies.

consider something like inline JSON instead of =, so if the cell text starts/ends with curly or square brackets, interpret as json or perhaps some other indicator like a dollar sign or pipe

consider extending the view json to specify that certain rows (such as _bottom) or cells are formatted a certain way such as all of column X (less the header) is currency, then we could get rid of the =$ construct and only support plain excel formulas.

Basically we need a way to replace the =$ with something more intuitive.

I think I like the view.json approach better, but it needs some development.

**Comment by JessicaMulein** [comment]
in progress.
!&&format&&=formula is now partially supported.
